Makefile

```diff
@@ -3,8 +3,6 @@ GOBUILD=go build -trimpath
 ARCH ?= $(shell uname -m)
 OS ?= $(shell uname)
 
-SERVERTARGETS ?= dev server
-SERVERPARAMS ?=$(filter-out $(SERVERTARGETS), $(MAKECMDGOALS))
```

Comment:
what if it's empty?

Reply:
tested it, I guess it just works :)

Reply:
yeah, I used this hack until I needed to supply more complicated params, so I just run it with
```
make SERVERPARAMS="--config=./server.yml"
```